src/assets/i18n/es.json

```diff
@@ -252,6 +254,10 @@
   "INPUT_EMAIL": "Por favor, introduzca su usuario",
   "INPUT_PASSWORD": "Por favor, introduzca su contraseña",
   "DNI": "DNI",
+  "ABSOLUTE_GRADES":"Puntuación absoluta",
+  "RELATIVE_GRADES":"Puntuación relativa",
+  "ALERT_RESULTS_1":"Resultados de los",
+  "ALERT_RESULTS_2":"alumnos de esta ECOE",
```

Comment:
@pasqual-botella  ¿Esta cadena puede hacerse en una sola? ¿Porque esta dividido en 2 cadenas?

Reply:
Esta partida para poder poner enmedio el número de alumnos evaluados. Ese mensaje se puede cambiar a algo mas relacionado o una explicación en el futuro.

Reply:
Para introducir variables en una cadena se usa la sintaxis de poner entre {{ var }} para sustituirlo por el valor en la llamada.

Por ejemplo: "ITEMS_DELETED": "{{num}} {{item_type}} Eliminados"

Puedes usarlo desde el .ts con this.translate.instant()
this.translate.instant('ITEMS_DELETED',{"num": delPromises.length, "item_type": this.translate.instant('STUDENTS')})

O desde la propia plantilla de esta forma:

{{'STATION_DEPENDS_OF_STATION_DESCRIPTION' | translate : {name: station.parentStation.name} }}

Reply:
Esa es la nomenclatura que estoy usando. Ahora mismo, esos 2 alerts se usan para construir la cadena 
"{{ 'ALERT_RESULTS_1' | translate }} {{students.total}} {{ 'ALERT_RESULTS_2' | translate }}", que se traduce a 
"Resultados de los X alumnos de esta ECOE".
¿Hay alguna forma de dejar una cadena con un espacio para escribir el numero X a la hora de escribirse por pantalla? Así podria dejar solo una frase en el es.json.

Reply:
Ya he subido los cambios con la cadena unida.